docker-compose.local.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
```

Comment:
Is there or will there need to be a docker-compose yml for other envs? Or are we not needing docker-compose for staging and prod?

Reply:
Currently we're doing all the build webpack work that's happening in the assets dockerfile as part of the build in the heroku Dockerfile. If this becomes a problem we can investigate other solutions. I don't think it's worth making a techdebt issue for yet.